Makefile

```diff
@@ -55,6 +55,7 @@ clean: ## Delete intermediate build artifacts
 .PHONY: test
 test: build ## Run unit tests
 	$(GO) test -race -cover ./...
+	$(GO) test -tags protolegacy ./internal
```

Comment:
Does it make sense to run all tests with `protolegacy` tag instead of just internal ones to verify all of protocompile w/ protolegacy support?

Reply:
+1, that sounds reasonable

Reply:
done